clang/include/clang/Sema/Sema.h

```diff
@@ -11650,6 +11650,33 @@ class Sema final : public SemaBase {
     CTAK_DeducedFromArrayBound
   };
 
+  struct CheckTemplateArgumentInfo {
+    explicit CheckTemplateArgumentInfo(bool PartialOrdering = false,
```

Comment:
This constructor doesn't make it completely better?  We still end up having 2 very similar bools (actually, looks like 3?) that are pretty easily confused.

I DO like the documentation and pulling the converted together into 1 though.  But it would be great if we had a better way to represent the bools that made construction less obtuse.